contact/src/main/kotlin/edu/stanford/spezi/contact/ContactContent.kt

```diff
@@ -38,17 +37,17 @@ import java.util.Locale
 /**
  * ContactView composable function to display contact information.
  *
- * @param contact The contact associated with the view.
+ * @param modifier A modifier to modify the inner composable.
  *
- * @sample ContactComposablePreview
+ * @sample ContactContentPreview
  *
  * @see Contact
  * @see ContactOptionCard
  * @see AddressCard
  */
 @OptIn(ExperimentalLayoutApi::class)
 @Composable
-fun ContactComposable(contact: Contact, modifier: Modifier = Modifier) {
+fun Contact.Content(modifier: Modifier = Modifier) {
```

Comment:
Anything against conforming Contact directly to ComposableContent? It is indeed a ui layer model as it already has ui properties such as image resource and string resources

Reply:
Nope, just that we would then need to put it in the file of the Contact itself and wouldn't have separated UI code from Compose stuff, but actually I'm fine with moving it, if you want.